packages/@okta/vuepress-site/docs/guides/email-customization/customize-email-templates/index.md

```diff
@@ -33,7 +33,7 @@ Email templates use common and unique [Expression Language (EL) variables](https
 
 #### Overview of what this EA feature enables
 
-The Early Access (EA) Enhanced Email Macros feature provides you with the functionality to customize the email templates with even greater flexibility than EL. It changes the underlying email templating engine from EL variables to [Velocity Templating Language (VTL)](https://velocity.apache.org/). This allows you to customize your org's email templates so that you can use enhanced conditional logic, all of the attributes in the Okta [User Profile object](/docs/reference/api/users/#profile-object), and some of the org attributes in these macros, as documented below.
+The Early Access (EA) Enhanced Email Macros feature provides you with the functionality to customize the email templates with even greater flexibility than EL. It changes the underlying email templating engine from EL variables to [Velocity Templating Language (VTL)](https://velocity.apache.org/). This allows you to customize your org's email templates so that you can use enhanced conditional logic, all of the attributes in the Okta [User Profile object](/docs/reference/api/users/#profile-object), and some of the org attributes in these macros, as documented below. All of the Okta default email templates are available for customization.
 
 ##### What this EA feature doesn't enable
```

Comment:
What this EA feature doesn't enable -> Previous customized email templates ?

Reply:
thanks, that's more clear. added a sentence "Previously customized email templates will not persist." to start the paragraph.

(fyi - for GA, we will do an auto migration, so previous emails will persist, and we can then remove this section